packages/lib-classifier/src/store/TutorialStore.spec.js

```diff
@@ -53,10 +75,10 @@ const authClientStubWithUser = {
 }
 
 describe('Model > TutorialStore', function () {
-  afterEach(function () {
-    clientStub.tutorials.get.resetHistory()
-    clientStub.tutorials.getAttachedImages.resetHistory()
-  })
+  // afterEach(function () {
+  //   clientStub.tutorials.get.resetHistory()
+  //   clientStub.tutorials.getAttachedImages.resetHistory()
+  // })
 
```

Comment:
Are these still needed? Should they be deleted?

Reply:
They should be deleted. Thanks.